tests/test_config.py

```diff
@@ -134,6 +147,16 @@ def test_migrate_v3_backup(tmp_path: Path, patch_rootdir: None):
     assert temp_config_file.with_suffix(".yaml.bak").read_text() == v3_config
 
 
+def test_migrate_v4(tmp_path: Path, patch_rootdir: None):
+    """Test migration from 4.0.0 to 4.0.1"""
+    temp_config_file = tmp_path / "temp_invokeai.yaml"
+    temp_config_file.write_text(v4_config)
+
+    conf = load_and_migrate_config(temp_config_file)
+    assert Version(conf.schema_version) >= Version("4.0.1")
```

Comment:
This was written so that the test would continue to work when we move to version 4.0.2 -- because the test is running on the "live" config. I'm reworking this as suggested so that the migrator is an instance, in which case the exact value check will work.